### ISSUE TITLE
For smooth loss func advise

### ISSUE THREAD

**Issue by Sparknzz** [xuebinqin/U-2-Net#251]
SInce I tried BCE loss for a while, the loss struggle around 0.03. I thought the BCE is not that suitable for smooth labeled training. As for lots of dataset, gts are labeled in [0-1] including DUTS-TR you used. So isn't it better to use mse loss, even KL Divergence as the loss function instead of BCE? Or convert it to be a ce problem with softmax activation. @xuebinqin

**Comment by rdutta1999** [comment]
For my background removal task, I had initially used BCE (with the ground truth masks having only 0 and 1) but the edges seemed to be too sharp for my taste. Using L2 loss, the edges turned out to be much smoother (with ground truth values being continuous in the range [0,1] ) and the generated probability map is much smoother; can be thought of as an alpha matte.
